src/app/modules/shared/components/input-date/input-date.component.html

```diff
@@ -1,10 +1,12 @@
 <input
   [value]="value"
   [disabled]="isDisabled"
-  (input)="onInput($event.target.value)"
-  type="date"
+  (dateInput)="onInput($event.target.value.format('YYYY-MM-DD'))"
```

Comment:
I think it is a better way, to apply the format in component.ts and use DATE_FORMAT